nextflow.config

```
@@ -86,104 +85,116 @@ try {
 }
 
 // Load nf-core/isoseq custom profiles from different institutions.
-// Warning: Uncomment only if a pipeline-specific instititutional config already exists on nf-core/configs!
-// try {
-//   includeConfig "${params.custom_config_base}/pipeline/isoseq.config"
-// } catch (Exception e) {
-//   System.err.println("WARNING: Could not load nf-core/config/isoseq profiles: ${params.custom_config_base}/pipeline/isoseq.config")
-// }
+try {
+    includeConfig "${params.custom_config_base}/pipeline/isoseq.config"
+} catch (Exception e) {
+    System.err.println("WARNING: Could not load nf-core/config/isoseq profiles: ${params.custom_config_base}/pipeline/isoseq.config")
+}
 profiles {
     debug {
-        dumpHashes             = true
-        process.beforeScript   = 'echo $HOSTNAME'
-        cleanup                = false
+        dumpHashes              = true
+        process.beforeScript    = 'echo $HOSTNAME'
+        cleanup                 = false
+        nextflow.enable.configProcessNamesValidation = true
     }
     conda {
-        conda.enabled          = true
-        docker.enabled         = false
-        singularity.enabled    = false
-        podman.enabled         = false
-        shifter.enabled        = false
-        charliecloud.enabled   = false
-        apptainer.enabled      = false
+        conda.enabled           = true
+        docker.enabled          = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        shifter.enabled         = false
+        charliecloud.enabled    = false
+        conda.channels          = ['conda-forge', 'bioconda', 'defaults']
```

```suggestion
        conda.channels          = ['conda-forge', 'bioconda']
```
We don't allow the `defaults` channel anymore due to licensing issues